lnwallet/channel.go

```diff
@@ -1324,8 +1327,8 @@ func (lc *LightningChannel) fetchCommitmentView(remoteChain bool,
 	// on its total weight. Once we have the total weight, we'll multiply
 	// by the current fee-per-kw, then divide by 1000 to get the proper
 	// fee.
-	totalCommitWeight := commitWeight + btcutil.Amount(htlcWeight*numHTLCs)
-	commitFee := (lc.channelState.FeePerKw * totalCommitWeight) / 1000
+	weight := commitWeight + btcutil.Amount(htlcWeight*numHTLCs)
```

Comment:
Hmm, downgraded to a less descriptive variable name? Was this only to ensure that the line below (with the added operation) wouldn't overflow 80 characters? If so, then either the line should be wrapped, or the `- dustFees` can mutate the `commitFee` variable on a new line. In either case, we end up with an additional line.